docs-website/docusaurus.config.js

```diff
@@ -69,6 +71,26 @@ module.exports = {
           label: "Roadmap",
           position: "right",
         },
+        {
+          type: "dropdown",
+          label: "v0.10.4",
+          position: "right",
+          items: [
+            {
+              label: "v0.10.3",
+              href: "https://docs-website-lgy6es9n7-acryldata.vercel.app/docs/",
```

Comment:
what's the plan for keeping these up to date? does this need to be part of our release process?

Reply:
I was thinking of linking these to specific vercel deployments that correspond to the last commit of each release - would that work?

Reply:
Yup, let’s just make sure to update our docs to include this step in the release process